job_spec/RTC_GAMMA.yml

```
@@ -88,14 +88,11 @@ RTC_GAMMA:
         type: boolean
   cost_profiles:
     EDC:
-      cost_parameter: resolution
+      cost_parameters: [resolution]
       cost_table:
-        - parameter_value: 30.0
-          cost: 5.0
-        - parameter_value: 20.0
-          cost: 15.0
-        - parameter_value: 10.0
-          cost: 60.0
+        30.0: 5.0
+        20.0: 15.0
+        10.0: 60.0
```

This is where the float vs. int issue might bite us, I'd suggest using int keys but we should test submitting jobs with both float and int values for `resolution` and see what happens.
```suggestion
        30: 5.0
        20: 15.0
        10: 60.0
```